.travis.yml

```diff
@@ -132,10 +132,41 @@ jobs:
         DOCKER_IMAGE_ID: debian-java
         JDK: 11
 
+    - name: "python on s390x"
+      os: linux
+      arch: s390x
+      env:
+        <<: *global_env
+        ARCH: s390x
+        ARROW_CI_MODULES: "PYTHON"
+        DOCKER_IMAGE_ID: python-sdist
```

Comment:
You should probably use `ubuntu-python` instead.

Reply:
Thank you!